### PR TITLE
fix shanbay and jike loader error

### PR DESCRIPTION
**Pull request by umm233**
1. 修复扇贝单词接口地址错误
2. 修复即刻 loader 由于参数格式变化导致生成失败
3. 修复 README 文档锚点错误

### CODE REVIEW

File: github_poster/loader/jike_loader.py [hunk 1]
```diff
@@ -48,13 +48,14 @@ def add_loader_arguments(cls, parser, optional):
...
             type=str,
+            required=True,
             help="The user id of jike",
         )
         parser.add_argument(
             "--jike_cookie",
             dest="jike_cookie",
             type=str,
-            required=optional,
+            required=True,
```
Comment (yihong0618) [inline github_poster/loader/jike_loader.py:58]: Change here maybe we can not support Multiple or summary.

**Review by yihong0618 — APPROVED** [review]
LGTM

File: github_poster/loader/jike_loader.py (hunk 1 above)
Comment (umm233) [reply github_poster/loader/jike_loader.py:58]: ok, I will change back.

**Comment by yihong0618** [comment]
And `CI` failed seems unused  problem
image

**Comment by yihong0618** [comment]
谢谢～

**Comment by umm233** [comment]
> 谢谢～

耶！

**Pull request merged**